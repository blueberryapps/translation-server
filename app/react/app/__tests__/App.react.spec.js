import { shallow } from 'enzyme';
import React from 'react';
import App from '../App.react';

test('shallowly renders App', () => {
  expect(shallow(<App />)).toMatchSnapshot();
});
