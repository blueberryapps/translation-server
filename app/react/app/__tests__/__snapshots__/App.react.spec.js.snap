exports[`test shallowly renders App 1`] = `
<h1>
  Haystack
</h1>
`;
